test/business-sync-custom-reports-spec.js

```diff
@@ -0,0 +1,291 @@
+var businessSyncSpecHelperMaker = require('./helpers/business-sync-spec-helper-maker.js');
```

Comment:
can just skim over these tests... I tested every scenario that I thought was important.